### ISSUE TITLE
Distal extension of skin lesion over time

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9963]
**Preferred term label:**
Distal extension of skin lesion over time

**Synonyms**
Peripheral

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Chromoblastomycosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
This HPO would not typically have terms that describe the temporal evolution of a feature, because this would lead to a combinatorial explosion. Instead, we would use a phenopacket. Admittedly, it is difficult for skin lesions, but I do not think this will work now.
